src/test/java/org/rulez/demokracia/pdengine/votecast/VoteCastValidationTest.java

```diff
@@ -54,76 +63,92 @@ public void cast_vote_does_not_delete_ballot_if_canVote_is_false() {
     List<RankedChoice> theCastVote = prepareCastVote(false);
 
     assertThrows(
-        () -> voteManager.castVote(adminInfo.voteId, ballot, theCastVote)
+        () -> castVoteService.castVote(
```

Comment:
This expression can be moved to a helper method, because used a couple of times.